lib/Mojolicious/Plugin/PODViewer.pm

```diff
@@ -295,22 +334,21 @@ __DATA__
           { module => $module, format => 'txt' } %>,
         <%= link_to 'CPAN' => 'plugin.podviewer', { module => $module } %>)
     </span>
-</div>
-
-<h1><a id="toc">CONTENTS</a></h1>
-<ul>
-    % for my $topic (@$topics) {
-        <li>
-            %= link_to splice(@$topic, 0, 2)
-            % if (@$topic) {
-                <ul>
-                    % while (@$topic) {
-                        <li><%= link_to splice(@$topic, 0, 2) %></li>
-                    % }
-                </ul>
-            % }
-        </li>
+</h1>
+<h2 class="d-print-none"><a id="toc">Contents</a></h2>
+<ul class="list-unstyled d-print-none">
```

Comment:
If a user wants these to not be printed or the list to not have bullets, they can do that themselves. If we want them not to be printed or not to have bullets by default, we can do that in CSS in the layout. By doing it manually in the layout it's more obvious what's happening when a user wants to make their own layout. So, we shouldn't change the classes on this `<h2>` and `<li>`. Also, the `<h2>` should be an `<h1>` because of the above change to the crumbs.